docs/walkthrough/using-functions.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Using Functions and Isolates
+nav_order: 12
+parent: Getting Started
+---
+
+## Using Functions and Isolates
+
+Functions exist in the following forms:
+
+- `subflow`
+- `isolated`
+- `reusable`
+- `knative-namespace`
+- `knative-global`
+
+This article demonstrates how each of the available function types is used within a workflow.
+
+### subflow
+
+A `subflow` function allows a workflow to execute another workflow. View [this article](/docs/walkthrough/subflows.html) for more detailed information.
+
+### isolated
+
+An 'isolated' function is similar to `reusable`, `knative-namespace`, and `knative-global` functions, but with some key differences in terms of implementation that are detailed [here](/docs/walkthrough/isolated-functions.html).
+
```

Comment:
I think a sentence or two explaining *why* someone might be interested in this type of function is probably needed. 

Something like:

Isolated functions are less performant than other types, but can be useful if you need greater isolation or just want to write your own function and prefer the simpler design pattern of interacting with the file-system instead of implementing a server that adheres to our reusable function spec.